cmd/backfillContractDiffs.go

```diff
@@ -0,0 +1,87 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/storage"
+	"github.com/makerdao/vulcanizedb/libraries/shared/storage/backfill"
+	"github.com/makerdao/vulcanizedb/utils"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var (
+	backfillContractDiffsAddress     string
+	backfillContractDiffsBlockNumber int64
+	backfillContractDiffsAddressFlag = "all-contract-values-address"
+	backfillContractDiffsBlockFlag   = "all-contract-values-block-number"
+)
+
+var backfillContractDiffsCmd = &cobra.Command{
+	Use:   "backfillContractDiffs",
+	Short: "Backfills all diffs for the given contract starting at the given block.",
+	Long: fmt.Sprintf(`Backfills diffs belonging to the given contract starting at the given block.
+
+Use: ./vulcanizedb backfillContractDiffs --config=<config.toml> --%s=<block number> --%s=<address>`, backfillContractDiffsBlockFlag, backfillContractDiffsAddressFlag),
```

Comment:
Couldn't find any easy way to come up with the contract's deployment block, so I just made so you have to pass it in :P